### PR TITLE
release 0.41.0

### PR DESCRIPTION
**Pull request by nithyatsu**
# Description

release 0.41.0

## Type of change

- This pull request fixes a bug in Radius and has an approved issue (issue link required).
- This pull request adds or changes features of Radius and has an approved issue (issue link required).
- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

Fixes: #issue_number

## Contributor checklist
Please verify that the PR meets the following requirements, where applicable:

- [ ] An overview of proposed schema changes is included in a linked GitHub issue.
- [ ] A design document PR is created in the [design-notes repository](https://github.com/radius-project/design-notes/), if new APIs are being introduced.
- [ ] If applicable, design document has been reviewed and approved by Radius maintainers/approvers.
- [ ] A PR for the [samples repository](https://github.com/radius-project/samples) is created, if existing samples are affected by the changes in this PR.
- [ ] A PR for the [documentation repository](https://github.com/radius-project/docs) is created, if the changes in this PR affect the documentation or any user facing updates are made.
- [ ] A PR for the [recipes repository](https://github.com/radius-project/recipes) is created, if existing recipes are affected by the changes in this PR.

### CODE REVIEW

File: docs/release-notes/v0.41.0.md
Comment (willtsai) [inline docs/release-notes/v0.41.0.md:11]:
```suggestion
## Highlights

### Improved UI for `rad init` CLI prompt in a small terminal window
We have resolved a usability issue where the text in the `rad init` CLI user interface gets cropped out of the visible area in a small terminal window. The text display area for the `rad init` UI now gets dynamically changed based on the terminal window dimensions, allowing for the text to be displayed instead of being cropped out of the visible display area.
```

File: docs/release-notes/v0.41.0.md
Comment (willtsai) [inline docs/release-notes/v0.41.0.md:13]:
```suggestion
```

File: .github/workflows/functional-test-cloud.yaml
Comment (sk593) [inline .github/workflows/functional-test-cloud.yaml:77]: We should make sure to cherry pick this into the release branch in another PR before merging this PR

File: .github/workflows/functional-test-cloud.yaml
Comment (kachawla) [inline .github/workflows/functional-test-cloud.yaml:77]: does this need to be rebased from main?

Comment (kachawla) [reply .github/workflows/functional-test-cloud.yaml:77]: actually nvm, just realized we needed to cherry pick this commit from main

**Review by kachawla — APPROVED** [review]
Looks good to me. Please get a PM approval as well for highlights before merging.

**Review by willtsai — APPROVED** [review]

**Pull request merged**
